device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsDeviceAuthenticationX509.java

```diff
@@ -98,4 +98,14 @@ protected Boolean isLinkFound(String linkName)
         // Codes_SRS_AMQPSDEVICEAUTHENTICATIONX509_12_012: [The function shall override the default behaviour and return true.]
         return true;
     }
+
+    /**
+     * X509 authentication does not have any links, so this will always return true
+     * @return true
+     */
+    @Override
+    public Boolean operationLinksOpened()
```

Comment:
Can this be private?

Reply:
No, but it can be package private. I'll edit it